src/sentry/static/sentry/app/components/forms/multiSelectField.jsx

```diff
@@ -34,19 +36,24 @@ export default class MultiSelectField extends InputField {
   render() {
     return (
       <MultiSelect
+        style={{width: 200, zIndex: 100, overflow: 'visible'}}
+        value={this.state.values}
+        {...this.props}
```

Comment:
Leave it in the last position in case we want to override any of the methods below

Reply:
I think (except for `arrowRender`), you shouldn't override the other props

Reply:
Most likely you wouldn't, but I'd rather just leave it fully extensible anyway

Reply:
At a minimum, you can't override `onChange`